compilation-config.yaml

```diff
@@ -21,6 +21,16 @@ fig_wg1_spm4:
   output_data_path: data/processed/fig-spm4/fig-spm4-timeseries.csv
   compile_function: openscm_ar6_wg1_data_compilation.compile_spm_fig_4_timeseries
 
+fig_wg1_spm7:
+  variables:
+      Cumulative Carbon Stored|Atmosphere:
+      Cumulative Carbon Stored|Land:
```

Comment:
Is it just me or does the config yaml still have 'Stored' even though the output has 'Uptake'?

Reply:
Thanks for catching that!